### PR TITLE
Sort the output of ProjectHealthConsoleReportBuilder on the printed suggestion

### PR DESCRIPTION
**Pull request by mvegter**

### CODE REVIEW

**Comment by mvegter** [comment]
The Advice#compareTo is not the natural sorting that I was aiming for, this fixes it by doing it the old way of sorting the (to) print advice.

**Review by autonomousapps — COMMENTED** [review]
At this point, I think this class should be tested so we can really know it works, and protect against regressions.

**Comment by autonomousapps** [comment]
Thanks for keeping your branch up to date @mvegter. Are you interested in writing a unit test to validate the change?

File: src/test/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilderTest.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  fun adviceOfRemoveShouldBeSorted() {
+    val dependencyAdvice = setOf(
+      Advice.ofRemove(ModuleCoordinates("com.project.a", "1.0"), "implementation"),
+      Advice.ofRemove(ModuleCoordinates("com.project.c", "1.0"), "api"),
+      Advice.ofRemove(ModuleCoordinates("com.project.b", "1.0"), "api"),
+    )
+    val projectAdvice = ProjectAdvice("dummy", dependencyAdvice, emptySet())
+
+    val consoleText = ProjectHealthConsoleReportBuilder(projectAdvice).text
+    Truth.assertThat(consoleText).isEqualTo("" +
```
Comment (autonomousapps) [inline src/test/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilderTest.kt:21]: nit: normally I prefer a static import of `Truth.assertThat`

**Review by autonomousapps — APPROVED** [review]
Thanks for the test!

**Pull request merged**